wcivf/apps/parishes/models.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+
+
+class ParishCouncil(models.Model):
```

Comment:
Small niggle with this that might be tricky to change now - this isn't modelling a council but a council's election, right?

I think `ParishCouncilElection` might be a better name, but if it's tricky to change then we can leave as is for now (this is the MVP after all)

Reply:
Yeah you are right - I thought this right at the start and then never went back to it. Will sort